### ISSUE TITLE
apt-get can not install dependencies on Ubuntu 17.10

### ISSUE THREAD

**Issue by lukasfischer83** [JuliaGraphics/QML.jl#51]
On ubuntu 17.10 I got:

> E: Unable to locate package qtdeclarative5-dialogs-plugin
> E: Unable to locate package qtdeclarative5-controls-plugin
> E: Unable to locate package qtdeclarative5-quicklayouts-plugin

Those seem to be transitional packages, the corresponding new ones I needed to install were

> sudo apt-get install qml-module-qtquick-controls qml-module-qtquick-dialogs 

Of course I had to comment the apt-get line in the build.jl script to get the install process done.

**Comment by barche** [comment]
OK, thanks, would the following command be complete enough, then? I don't have a Ubuntu 17 system to test on:
```
sudo apt-get install cmake cmake-data qtdeclarative5-dev qml-module-qtquick-controls qml-module-qtquick-dialogs qmlscene qt5-default
```

**Comment by Charles-Schleich** [comment]
Hello ! 
I was having the same issue on Ubuntu 18.04 as @lukasfischer83, but your commented sudo apt-get fixed it. 
Thanks @barche !

**Comment by ufechner7** [comment]
Perhaps this can be closed now that 0.8 is released?
